modules/ROOT/pages/design-add-api-dependency.adoc

```diff
@@ -61,13 +61,36 @@ _Result_: The rulesets are added to your project.
 When you have your branch, a pop-up appears where you can see the missing ruleset filtered by tags, category, and lifecycle state. To edit those filters, click *Edit criteria*. When you are done, click *Add all rulesets*. +
 _Result_: The rulesets are added to your project.
 
+== Improving Ruleset Validation
 
-[fix-gov-issues]
+You can validate your rulesets manually or automatically:
+
+* To turn manual validation on, click *Validate for Governance* in the *Governance* tab of the *Project Errors* section of the text editor. 
```

Comment:
Actually, you need to enable manual mode first by clicking on the Green auto-validation button toggle to switch it off. Then, the "Validate for Governance" button is enabled and user can trigger validation when clicking the button.